src/Pages/About/About.js

```diff
@@ -1,37 +1,99 @@
 import React from 'react';
+import {
+  Container,
+  Card,
+  CardBody,
+  CardText,
+  Row,
+  Col,
+} from 'reactstrap';
 
 import './About.css';
 
 export default function AboutPage() {
-  /* eslint-disable max-len */
-  const photos = [
-    'https://user-images.githubusercontent.com/36345325/183006899-43dd8a1d-748c-414d-9643-81fc420d6f6a.png',
-    'https://user-images.githubusercontent.com/36345325/183006944-9ac31633-3651-41e7-95f2-36470cd2fb79.png',
-    'https://user-images.githubusercontent.com/36345325/183007229-cf4719cb-c608-465d-a586-22a01c9d402b.png',
-    'https://user-images.githubusercontent.com/36345325/183007349-2717ab80-ab83-4f24-8c54-35d8fc99b945.png',
-    'https://user-images.githubusercontent.com/36345325/183007397-903b28d0-7519-4c09-a820-54884c385108.png',
-    'https://user-images.githubusercontent.com/36345325/183007498-b9a87fc6-8f62-47bb-8518-4dfefa4c1060.png'
-  ];
-  /* eslint-enable max-len */
-
   return (
     <>
       <div className='about-bg'>
-        <div className='about-header'>
-          <h1> The Next Frontier of Innovation at San Jose State</h1>
-        </div>
-        <div className='about-text'>
-          <h3>
-            The Software and Computer Engineering Society aims
-            to provide students with sense of community, industry-relevant
-            experience and mentorship.
-          </h3>
-        </div>
-        <div className='about-flexbox-img'>
-          {photos.map((photo, index) => (
-            <img className='img-size' src={photo} alt="scephoto" />
-          ))}
-        </div>
+        <Container>
+          <Row className='center about-text'>
+            <Row className='center'>
+              <Col className='text-col'>
+                <h1> The Next Frontier of Innovation at San Jose State.</h1>
+                <p>
+                  The Software and Computer Engineering Society aims
+                  to provide students with sense of community, industry-relevant
+                  experience and mentorship.
+                </p>
+              </Col>
+              <Col>
+                <Card style={{ minWidth: '40vw' }}>
+                  <CardBody>
+                    <CardText>
+                      Network with peers in similar majors and classes
+                    </CardText>
+                  </CardBody>
+                </Card>
+                <Card style={{ minWidth: '40vw' }}>
+                  <CardBody>
+                    <CardText>
+                      Visit company campuses, attend company info sessions and club events
+                    </CardText>
+                  </CardBody>
+                </Card>
+                <Card style={{ minWidth: '40vw' }}>
+                  <CardBody>
+                    <CardText>
+                      Set yourself apart from others by learning skills and resume help
```

Comment:
I feel like there's a grammar issue with "and resume help" maybe say, "receiving resume support"